great_expectations/data_context/types/base.py

```diff
@@ -1449,6 +1510,10 @@ class Meta:
     validations = fields.List(cls_or_instance=fields.Dict())
     profilers = fields.List(cls_or_instance=fields.Dict())
 
+    @post_load
```

Comment:
We must not use @post_load for CheckpointConfig classes, because it extends BaseYamlConfig, and for YamlConfig classes we use `from_commented_map()` as part of `serialize`; so upon deserialization, we do not want the object, we want a dictionary -- to instantiate the given Config class.

Reply:
Okay - removed.